definitions/divorce/json/AuthorisationCaseEvent.json

```diff
@@ -4385,7 +4385,7 @@
     "LiveFrom": "01/03/2019",
     "CaseTypeID": "DIVORCE",
     "CaseEventID": "solicitorApplyForDN",
-    "UserRole": "[PETSOLICITOR]",
+    "UserRole": "caseworker-divorce-solicitor",
```

Comment:
Reverting the UserRole changes introduced previously, until further info about solicitor journey